server.js

```diff
@@ -48,13 +47,12 @@ app.get("/api/random-restaurants", async (req, res) => {
       .status(500)
       .json({ message: "Internal Server error", dsc: error.message });
   }
-  res.sendFile(path.join(__dirname, "client/build", "index.html"));
 });
 
 // Handle React routing, return all requests to React app
 app.get("*", (req, res) => {
-  console.log(req);
-  res.sendFile(path.join(__dirname, "client/build", "index.html"));
+  console.log(req, res);
```

Comment:
also remove that console.log ;)

Reply:
I only have that `console.log` so that the warning of my linter doesn't react all the time :D do you have any other idea to solve this warning?

Reply:
hmm I don't get that warning but als my build folder is in .eslintignore- maybe that helps?